src/containers/SubscriberSignup/index.js

```diff
@@ -36,13 +36,27 @@ const options = [...statesOpts, ...districtOpts, ...stateEvents]
 
 class SubscriberSignup extends React.Component {
 
+    state = {
+        emailDataSource: [],
+        submitText: 'Add New',
+    }
```

Comment:
since we are using redux, this should be in app state, not component state

Reply:
I've moved these to the subscriber branch. 

Particularly with the button text, it seems weird to me to have things in the subscribers data that are really only relevant for the display of a single component, but I suppose this is the accepted convention on how these things should be structured...? Or is there a better place to put it?